api-report/sequence.api.md

```diff
@@ -136,13 +136,28 @@ export interface IInterval {
 export interface IIntervalCollection<TInterval extends ISerializableInterval> extends TypedEventEmitter<IIntervalCollectionEvent<TInterval>> {
     // (undocumented)
     [Symbol.iterator](): Iterator<TInterval>;
+    // @deprecated
     add(start: number, end: number, intervalType: IntervalType, props?: PropertySet, stickiness?: IntervalStickiness): TInterval;
+    add({ start, end, props, stickiness, }: {
+        start: number;
+        end: number;
+        props?: PropertySet;
+        stickiness?: IntervalStickiness;
+    }): TInterval;
     // (undocumented)
     attachDeserializer(onDeserialize: DeserializeCallback): void;
     // (undocumented)
     readonly attached: boolean;
     attachIndex(index: IntervalIndex<TInterval>): void;
+    // @deprecated
     change(id: string, start?: number, end?: number): TInterval | undefined;
+    change({ id, start, end, props, }: {
+        id: string;
+        start?: number;
```

Comment:
@scarlettjlee changed this method to make start/end required parameters on next. That will be a little harder to express with the object signature, but still doable. You could either use separate overloads or a technique like [this](https://www.typescriptlang.org/play?#code/LAKAlgdgLgpgTgMwIYGMYAIAKcD2AHeKATwGUYp0BvUASAG0BrGIgLnQGco5IBzAXTZIIRANygAvqFCRYiVBkw52YKGBwQSBFFVqckcKGwgBXALYAjeGJA0YEACZGzluNckhp0eMjRZcBAyIAERgAGygkTRhtahs8f3YAfjZsfEJScgkpcC85XwBBc3Y7KCiY3QiDZPQIGAA3K1o7e2rahtcsjxBiAnR8uB4AFSJegF50AApFZVV1MvQAHz6ikrKASnQAMj80wJDwyK1rUBR1TnQEHBw2fqGRjHHYmj0DNgBWABomhzYAZgljiBThBzpccAAmG4DYZjHQ2F6GdCfb6OdC-L5xBJsSjoJBscG-dDucSA4Hncz6KF3WFPBHvDE0eL4djYi5XfEABiJAOyZIoFIAXlSYQ8qCTeWd+UgBZC+tD7uhHrQmXgWVQajAAO6pPBsADkeD13JA4q6nlkPgwAElcnUkKEAMI4UKhaKzCBwmgoAAWQh4MAmYFRnG4EB4H1xAzVtxFazYdRwQbcQA)

Reply:
since the two branches are converging pretty soon (this week IIRC), it's probably easiest to just do this immediately on main and wait til after the convergence point to merge them.